Source/Core/Color.js

```diff
@@ -141,12 +143,10 @@ define([
      */
     Color.fromAlpha = function(color, alpha, result) {
         //>>includeStart('debug', pragmas.debug);
-        if (!defined(color)) {
-            throw new DeveloperError('color is required');
-        }
-        if (!defined(alpha)) {
-            throw new DeveloperError('alpha is required');
-        }
+
+        Check.defined('color', color);
```

Comment:
Look out for this with the rest of the checks. The one exception is for arrays which should continue to be checked with `Check.defined` since we don't have a `Check.typeOf.array`.